### PR TITLE
Feature/activities list

### PR DESCRIPTION
**Pull request by NikCrmr**

### CODE REVIEW

File: activities.json [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    "emoji": "😃",
+    "description": "hold your smile for 10 seconds and uplift your mood",
+    "forEmotion": ["sadness", "anger", "fear"],
+    "tool": true,
+    "link": "/smiletrainer"
+  },
+  {
+    "title": "Inspirational Quotes",
+    "emoji": "🌟📖",
+    "description": "hold your smile for 10 seconds and uplift your mood",
```
Comment (MerleMak) [inline activities.json:13]: I already reviewed this 🤔
Why is it in several PRs?

File: components/Layout/Layout.js [hunk 2]
```diff
@@ -19,8 +19,20 @@ export const Page = styled.div`
 
 export const Navigation = styled.div`
   position: fixed;
+  justify-content: center;
+  align-items: center;
   bottom: 0;
   display: flex;
   gap: 20px;
-  margin-bottom: 60px;
+  /* margin-bottom: 60px; */
```
Comment (MerleMak) [inline components/Layout/Layout.js:27]:
```suggestion
```

File: components/ActivitiesForm/ActivitiesForm.js [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+  const [showPicker, setShowPicker] = useState(false);
+
+  function onEmojiClick(emoji) {
+    setInputString((prevInput) => [...prevInput, emoji]);
+    setShowPicker(false);
+    window.scrollTo(0, 0);
+  }
+
+  useEffect(() => {
+    console.log("inputString", inputString);
```
Comment (MerleMak) [inline components/ActivitiesForm/ActivitiesForm.js:21]:
```suggestion
```

**Review by MerleMak — COMMENTED** [review]
I feel like most of this I already reviewed in the last PR?
Only the emoji picker was new i think?
Please make sure to package the PRs in a way that its only reviewed once. 

But the code looks good!

File: activities.json (hunk 1 above)
Comment (janphilipp-winkler) [reply activities.json]: 😇

**Pull request merged**